runhouse/main.py

```diff
@@ -479,6 +477,7 @@ def status(
             "\N{smiling face with horns} Runhouse Daemon is not running... \N{No Entry} \N{Runner}"
         )
         return
+
     return _print_status(cluster_status, current_cluster)
```

Comment:
i know not part of this PR, but can we update the `print_status` docstrings to use google format similar to the rest of the docs? also does it need to return anything? not sure we're doing anything with `status_data`